mne/viz/evoked.py

```diff
@@ -890,6 +896,13 @@ def plot_evoked_white(evoked, noise_cov, show=True):
                               scalings=None, rank=None, show=show)
 
 
+def _match_proj_type(proj, ch_names):
+    """see if proj should be counted"""
+    proj_ch_names = proj['data']['col_names']
+    select = any(kk in ch_names for kk in proj_ch_names)
+    return select
```

Comment:
what is this doing? don't you need to only count the projs that have been applied?